### PR TITLE
Fixes from QA

### PR DESCRIPTION
**Pull request by jalavik**
Assortement of minor fixes that was needed to properly run QA.

### CODE REVIEW

File: inspirehep/modules/search/views.py [hunk 1]
```diff
@@ -69,6 +69,12 @@ def search():
                            **ctx)
 
 
+@blueprint.route("/ping")
+def ping():
+    """Ping for load balancer."""
+    return "OK", 200
```
Comment (jacquerie) [inline inspirehep/modules/search/views.py]: Uh? Who is looking for this guy under `/search/ping`? It should be pointing to `/ping`...

Comment (jalavik) [inline inspirehep/modules/search/views.py]: The url_prefix for the blueprint is `/` so it will be `/ping`.

Comment (jacquerie) [inline inspirehep/modules/search/views.py]: Whoops, you're right about the `url_prefix`. On the other hand, Isn't this guy already defined? https://github.com/inspirehep/inspire-next/blob/644530717f503b4913f58336d7b5246088fbac24/inspirehep/modules/theme/views.py#L172-L174

Maybe we just need to add an explicit 200 to that?

Comment (jalavik) [inline inspirehep/modules/search/views.py]: Ah yeah, you are right!

**Comment by jacquerie** [comment]
LGTM, rebase and merge?

File: requirements.txt [hunk 2]
```diff
@@ -22,6 +22,9 @@
 # waive the privileges and immunities granted to it by virtue of its status
 # as an Intergovernmental Organization or submit itself to any jurisdiction.
 
+# Raven backported fix for new Flask Login on old Sentry
```
Comment (lnielsen) [inline requirements.txt]: FYI https://github.com/zenodo/zenodo/blob/master/zenodo/config.py#L197-L198

Comment (lnielsen) [inline requirements.txt]: + https://github.com/zenodo/zenodo/blob/master/setup.py#L132

Comment (jalavik) [inline requirements.txt]: Thanks @lnielsen!

**Comment by jacquerie** [comment]
:ship:

**Pull request merged**